pkgs/applications/editors/vscode/generic.nix

```diff
@@ -68,7 +68,12 @@ let
 
     runtimeDependencies = lib.optional stdenv.isLinux [ (lib.getLib systemd) fontconfig.lib libdbusmenu ];
 
-    nativeBuildInputs = [ unzip ] ++ lib.optionals stdenv.isLinux [ autoPatchelfHook nodePackages.asar wrapGAppsHook ];
+    nativeBuildInputs = [ unzip ]
+      ++ lib.optionals stdenv.isLinux [
+        autoPatchelfHook
+        nodePackages.asar
+        (wrapGAppsHook.override { makeBinaryWrapper = makeWrapper; })
```

Comment:
This kinda of looks the contrary of what I expect. The argument for `wrapGAppsHook` should be named `makeWrapper`, then set `makeWrapper = makeBinaryWrapper` in `all-packages.nix` and the override it here with `{ inherit makeWrapper; }`  instead.

Reply:
I agree. I'm making a big "clean up" PR for `makeBinaryWrapper`, I'll include that.

Reply:
I am going to say that this is a minor nitpick though. I am not against merging this as-is. It just looks strange.

Reply:
https://github.com/NixOS/nixpkgs/pull/172366